dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.34/V2_34_7__Create_Audit_Table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE audit (
```

Comment:
Perhaps make a few fields NOT NULL for integrity purposes?

Reply:
This table is still WIP, and the point of it all.. we will discuss it in second week of December (but yes, I agree 100%)